Timezone, openweathermap language, Readme

xremix opened the pull request:
- New Timezone Plugin
- openweathermap language feature
- In the openweathermap add on I now use the description as the "main" won't get translated.

manojlds commented on Time/timezone.5s.sh:
```diff
@@ -0,0 +1,13 @@
+
+# BitBar timezone plugin
+#
+# By Toni Hoffmann
```
Can you add plugin metadata as per the README?

manojlds commented on Weather/OpenWeatherMap/weather.15m.py:
```diff
@@ -1,13 +1,12 @@
-# <bitbar.dependencies>python</bitbar.dependencies>
+# Weather
+#
+# by Daniel Seripap (daniel@seripap.com)
```
Why are you removing the metadata?

xremix commented:
@manojlds not sure where the removal from the meta data was coming from as I just copied and pasted the plugin to my plugin folder.

Anyway I fixed this and added meta data to my plugin.
